build/entrypoint.sh

```diff
@@ -1,15 +1,12 @@
 #!/bin/bash
 
-# If we are using podman or docker compose use the repo config.toml
-if [[ -f /podman_compose/kai/config.toml ]]; then
-	cp /podman_compose/kai/config.toml /kai/kai/config.toml
-	sed -i "s/^host =.*/host = \"${POSTGRESQL_HOST}\"/g" /kai/kai/config.toml
-	sed -i "s/^database =.*/database = \"${POSTGRESQL_DATABASE}\"/g" /kai/kai/config.toml
-	sed -i "s/^user =.*/user = \"${POSTGRESQL_USER}\"/g" /kai/kai/config.toml
-	sed -i "s/^password =.*/password =\"${POSTGRESQL_PASSWORD}\"/g" /kai/kai/config.toml
+# If a custom config is specified, use it
+if [[ -f /podman_compose/build/config.toml ]]; then
+	printf "Using custom config.toml\n"
+	cp /podman_compose/build/config.toml /kai/kai/config.toml
```

Comment:
Will an end user running `podman compose up` have access to the `config.toml` on their local disk so they can see the settings and adjust?

The usecase we want to satisfy is to allow the end user to:
- View the `config.toml` from local disk
- Change settings on local disk to modify configured models

I'm not sure from a quick review of this PR if there is a mechanism for the end user to view/edit the `config.toml` that is used.  My current impression is we are copying it into the container at build time.  I see how that helps for running with the default configurations and I see the intent to override via environment variables, but unsure about how a user can modify the config.toml.

Reply:
The user can easily adjust their `config.toml`. The way it works in this PR is:

1. When the container is _built_, the `config.toml` that is present in the repo is copied over to the container
2. When the container is _run_, it first checks if the file `build/config.toml` exists. If it does, it copies it over to the container.

Thus, the user can specify their config via `build/config.toml`. There is an `example_config.toml` also present in that directory. We could even add `example_stable_config.toml` or something so the user has a config that can match their version. We can change the location that `entrypoint.sh` looks for if it makes more sense for the user to specify their custom config elsewhere.

Reply:
I updated how this works to make more sense. See my comment below.